### ISSUE TITLE
Errors in Firefox browser console when recording

### ISSUE THREAD

**Issue by Omniscimus** [mozilla/voice-web#397]
### Browsers
- Firefox 54.0.1 (64-bit)
- Firefox 55.0b13 (64-bit)

Does NOT occur in Chrome 59.0.3071.115 (Official Build) (64-bit)

### Operating systems
- macOS 10.12.6

### Description

While recording in Firefox, some errors pop up in the console. Everything on the pages seems to be working fine. In the Network tab of the developer tools, it looks like the recordings do get uploaded.

When navigating to [https://voice.mozilla.org/record](https://voice.mozilla.org/record), I get the following error 3 times: (link points towards [https://voice.mozilla.org/record](https://voice.mozilla.org/record))
`Invalid URI. Load of media resource  failed. record`

When clicking 'submit' after recording three sentences: (links are: [upload:1:1](https://voice.mozilla.org/upload/), [record](https://voice.mozilla.org/record), [demographic:1:1](https://voice.mozilla.org/upload/demographic/))
```
XML Parsing Error: syntax error
Location: https://voice.mozilla.org/upload/
Line Number 1, Column 1:  upload:1:1

Invalid URI. Load of media resource  failed. record

XML Parsing Error: syntax error
Location: https://voice.mozilla.org/upload/
Line Number 1, Column 1:  upload:1:1

Invalid URI. Load of media resource  failed.  record

XML Parsing Error: syntax error
Location: https://voice.mozilla.org/upload/
Line Number 1, Column 1:  upload:1:1

XML Parsing Error: syntax error
Location: https://voice.mozilla.org/upload/demographic/
Line Number 1, Column 1:  demographic:1:1
```

Is there any way for me to debug this without setting up an Amazon S3 account for testing? If not, how do I set up one? I noticed that they have some [free plans](https://aws.amazon.com/free/), which one would I need? If setting up a local test server is all too much of a hassle then that's fine, I'll work on other issues.

**Comment by mikehenrty** [comment]
I don't think you need an s3 account to test and debug this. The problem is not with s3, but the fact that the response is being interpreted by the browser as HTML, and instead we are sending text. I think we just need to set the right response header in nodejs ([here](https://github.com/mozilla/voice-web/blob/master/server/src/lib/clip.ts#L200) for instance).

If you would like to set up s3 for local testing, I do believe you can use a free plan, and then add aws creds like this:
http://docs.aws.amazon.com/AWSImportExport/latest/DG/SaveCredentials.html

**Comment by Omniscimus** [comment]
The XML errors are fixed by #411, but the `Invalid URI` errors still need to be fixed

**Comment by mikehenrty** [comment]
Looks like @Omniscimus fixed everything here! Thank you!!!
